src/navigation/components/NavigationItem/components/ContentWrapper/ContentWrapper.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../../../../../styles/variables';
+
+$block: '.#{$ns}content-wrapper';
+
+#{$block} {
+    &__icon {
+        display: flex;
+        width: 20px;
```

Comment:
We have a component Icon with size prop. Can we use it here?

Reply:
answer in https://github.com/gravity-ui/page-constructor/pull/159#discussion_r1118386508